Master build is broken due to checkstyle errors

**Problem description**
Merge of pull request #1518 introduced a few checkstyle errors:
```
:controller:checkstyleTest[ant:checkstyle] [ERROR] /home/travis/build/pravega/pravega/controller/src/test/java/io/pravega/controller/rest/v1/StreamMetaD
ataTests.java:12:8: Unused import - com.fasterxml.jackson.core.type.TypeReference. [UnusedImports]
[ant:checkstyle] [ERROR] /home/travis/build/pravega/pravega/controller/src/test/java/io/pravega/controller/rest/v1/StreamMetaD
ataTests.java:712:121: '{' is not preceded with whitespace. [WhitespaceAround]
[ant:checkstyle] [ERROR] /home/travis/build/pravega/pravega/controller/src/test/java/io/pravega/controller/rest/v1/StreamMetaD
ataTests.java:712:122: '{' is not followed by whitespace. [WhitespaceAround]
[ant:checkstyle] [ERROR] /home/travis/build/pravega/pravega/controller/src/test/java/io/pravega/controller/rest/v1/StreamMetaD
ataTests.java:712:122: '}' is not preceded with whitespace. [WhitespaceAround]
```

**Problem location**
Controller.

**Suggestions for an improvement**
Fix checkstyle errors.

--- a/controller/src/test/java/io/pravega/controller/rest/v1/StreamMetaDataTests.java
+++ b/controller/src/test/java/io/pravega/controller/rest/v1/StreamMetaDataTests.java
@@ -9,7 +9,6 @@
  */
 package io.pravega.controller.rest.v1;
 
-import com.fasterxml.jackson.core.type.TypeReference;
 import io.pravega.controller.server.rest.RESTServer;
 import io.pravega.controller.server.rest.RESTServerConfig;
 import io.pravega.controller.server.rest.impl.RESTServerConfigImpl;
@@ -709,7 +708,8 @@ public class StreamMetaDataTests {
         Response response = client.target(resourceURI).queryParam("from", fromDateTime).
                 queryParam("to", toDateTime).request().buildGet().invoke();
         assertEquals("Get Scaling Events response code", 200, response.getStatus());
-        final List<ScaleMetadata> scaleMetadataListResponse = response.readEntity(new GenericType<List<ScaleMetadata>>(){});
+        final List<ScaleMetadata> scaleMetadataListResponse = response.readEntity(
+                new GenericType<List<ScaleMetadata>>() { });
         assertEquals("List Size", 3, scaleMetadataListResponse.size());
         scaleMetadataListResponse.forEach(data -> {
             log.warn("Here");

